cmd/edit/firmware_set.go

```diff
@@ -85,6 +109,7 @@ func init() {
 		log.Fatal(err)
 	}
 
-	cmdFlags.StringVar(&editFWSetFlags.FirmwareUUIDs, "remove-firmware-uuids", "", "UUIDs of firmware to be removed from the set")
+	cmdFlags.StringVar(&editFWSetFlags.RemoveFirmwareUUIDs, "remove-firmware-uuids", "", "UUIDs of firmware to be removed from the set")
+	cmdFlags.StringVar(&editFWSetFlags.AddFirmwareUUIDs, "add-firmware-uuids", "", "UUIDs of firmware to be added to the set")
```

Comment:
This can be replaced with `https://pkg.go.dev/github.com/spf13/pflag#StringSliceVar` so then the `strings.Split()` isn't necessary

Reply:
Fixed in [4e3ec2e](https://github.com/metal-toolbox/mctl/pull/19/commits/4e3ec2e92d03187f8d351b99af644992bc386514)